stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -143,14 +145,21 @@ interface StripeIntent : StripeModel {
      * depends on this property to invoke authentication flows. The shape of the contents is subject
      * to change and is only intended to be used by the Stripe SDK.
      */
-    data class SdkData internal constructor(internal val data: Map<String, *>) {
-        internal val type: String = data[FIELD_TYPE] as String
+    data class SdkData internal constructor(
```

Comment:
Can this be marked as `@Parcelize`?

Reply:
sure, but this is the class that I'm trying to deprecate